protostar/commands/test/testing_seed.py

```diff
@@ -0,0 +1,26 @@
+import os
+from contextlib import contextmanager
+from contextvars import ContextVar
+from typing import Optional
+
+Seed = int
+_testing_seed: ContextVar[Seed] = ContextVar("testing_seed")
+
+
+@contextmanager
+def testing_seed(seed: Optional[Seed] = None):
+    if seed is None:
+        seed = int.from_bytes(os.urandom(4), byteorder="little", signed=False)
+
+    # HACK: Keep typechecker happy
```

Comment:
Can we remove this comment? We use it a lot, so there is no point mentaioning it as a hack (which technically is)